streaming/java/streaming-common/src/main/java/io/ray/streaming/common/utils/EnvUtil.java

```diff
@@ -0,0 +1,282 @@
+package io.ray.streaming.common.utils;
+
+import com.google.common.collect.Sets;
+import com.sun.jna.NativeLibrary;
+import io.ray.runtime.util.BinaryFileUtil;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.management.ManagementFactory;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EnvUtil {
+
+  private static final Logger LOG = LoggerFactory.getLogger(EnvUtil.class);
+
+  public static final String DEV_ENV_VALUE = "dev";
```

Comment:
also can be removed

Reply:
Then use what cluster name if started as local mode, this name is needed to build `WorkerRuntimeInfo`